pkg/daemon/daemon.go

```diff
@@ -486,6 +511,11 @@ func (dn *Daemon) completeDrain() error {
 	return nil
 }
 
+func (dn *Daemon) discoverPlatform() error {
```

Comment:
dn.platform is discovered and passed by operator controller when daemon.New is called, do we need this function in daemon?

Reply:
We do not and I thought I had removed it...